### PR TITLE
Use yao instead of fog

### PR DESCRIPTION
**Pull request by ku00**
This PR replaces fog with yao.

### CODE REVIEW

**Comment by udzura** [comment]
The code looks very good to me :+1: 

I'm going to test this in real project soon!

File: kakine.gemspec [hunk 1]
```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'fog'
+  spec.add_dependency "yao", "~> 0.1.0"
```
Comment (udzura) [inline kakine.gemspec]: yao's current version is `0.2.7`, so I hope here to use `~> 0.2.0` :exclamation:

File: lib/kakine/config.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+require 'yao'
+require 'yaml'
+
+module Kakine
+  class Config
+    def self.setup
```
Comment (udzura) [inline lib/kakine/config.rb:6]: `Kakine::Config.setup` must be explicitly called when client is invoked!

May be @ `lib/kakine/cli.rb`

**Comment by ku00** [comment]
Thank you!! :baby_chick:

**Comment by ku00** [comment]
Oh, Kakine command causes `Faraday::TimeoutError` when `Kakine::Config.setup` called @ `lib/kakine.rb`...

**Comment by udzura** [comment]
This works for me!!!

**Comment by udzura** [comment]
LGTM and CI is passing. I will merge!
Thanks @ku00 <3

**Pull request merged**